Cleanup and comments.

siegel opened the pull request:
bits of logic.

*	Rearranged variables and initializations at the top of the file.

*	Factored out the CSV ingestion code to the top of the file.

*	Adjusted some loop bodies to use "continue" for readability.

*	When viewing a single trip, the bus loop now stops after
finding the appropriate trip, since there are not going to
be any other trips with that ID in the data feed.

*	Normalized indentation where appropriate.

siegel committed:
*	Wrote comments to annotate code flow and the more complicated
	bits of logic.

*	Rearranged variables and initializations at the top of the file.

*	Factored out the CSV ingestion code to the top of the file.

*	Adjusted some loop bodies to use "continue" for readability.

*	When viewing a single trip, the bus loop now stops after
	finding the appropriate trip, since there are not going to
	be any other trips with that ID in the data feed.

*	Normalized indentation where appropriate.

kerri-hicks reviewed (changes requested):
These changes look great. Also, pie. One thought...Abstracting the csv data for trips feed and stops feed now requires data analysis for all page loads (including the main landing page and "about" page -- neither of which use it). I'd prefer to only call and ingest those files if they need to be used.

siegel committed:
*	Factored out the code which ingests the trips and stops feed
	data into separate functions, which can be called as needed
	rather than unconditionally doing the work (which is unnecessary
	for certain pages).

siegel commented:
I have addressed this in f62df4cf2a511d468222de6043cb1c660c7abc70. I have no idea whether your PR is updated with the change, or what. :-)
